### PR TITLE
cite: improves design in the modal box

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
* Implements download functionality.

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

File: inspire/modules/formatter/views.py [hunk 1]
```diff
@@ -47,3 +47,28 @@ def get_latex(recid, latex_format):
     record = get_record(recid)
     latex = Latex(record, latex_format).format()
     return jsonify({"result": latex, "recid": recid})
+
+
+@blueprint.route("/download-bibtex/<int:recid>")
+def get_bibtex_file(recid):
+    record = get_record(recid)
+    results = Bibtex(record).format()
+    generator = (cell for row in results
```
Comment (jmartinm) [inline inspire/modules/formatter/views.py]: What is the format returned by `Bibtex(record).format()` ? It does not return the text ready to be returned?

Comment (ioannistsanaktsidis) [inline inspire/modules/formatter/views.py]: It returns the text ready, so if you download it and open it looks like
this:
[image: Ενσωματωμένη εικόνα 1]

2015-09-02 13:08 GMT+02:00 Javier Martin Montull <notifications@github.com>:

Comment (jmartinm) [inline inspire/modules/formatter/views.py]: I think images don't work through email reply ;)

Comment (ioannistsanaktsidis) [inline inspire/modules/formatter/views.py]: Strange :/ in my machine is shows that is sent correctly! Nevermind, I am
 attaching you the bibtex file. If you open it with sublime you can see the
bibtex printed as we want.

**Comment by ioannistsanaktsidis** [comment]
@jmartinm  Rebased!

File: inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl [hunk 2]
```diff
@@ -106,18 +106,19 @@
...
+                <button type="button" class="btn btn-primary dropdown-toggle" id="btn-drop" data-toggle="dropdown" aria-expanded="false">
+                  Format: <span id="format{{record['recid']}}"></span> <span class="caret"></span>
+                </button>
+                <ul class="dropdown-menu" role="menu">
+                  <li><a class="pointer bibtex" id="bibtex{{record['recid']}}" data-recid="{{record['recid']}}">BibTex</a></li>
+                  <li><a class="pointer latex_eu" id="latex_eu{{record['recid']}}" data-recid="{{record['recid']}}">LaTex(EU)</a></li>
+                  <li><a class="pointer latex_us" id="latex_us{{record['recid']}}" data-recid="{{record['recid']}}">LaTex(US)</a></li>
+                </ul>
+              </div>
+              <a type="button" id="download{{record['recid']}}" class="btn btn-primary">Download</a>
```
Comment (jmartinm) [inline inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: We should use:

```python
record['control_number']
```

As I have found some problems using the field `recid`. If you check on ES, you will see that the field with the record id is called in that way.

Comment (jmartinm) [inline inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: Please, take care of the above comment in this and the other open PRs.

Comment (ioannistsanaktsidis) [inline inspire/base/templates/format/record/Inspire_Default_HTML_brief.tpl]: I made the changes and pushed them. The PRs are ready.

**Comment by jmartinm** [comment]
Merged 31ccf27b1e223f8e647f2a2844294b02b5234375

**Pull request closed without merging**